model/project_parser.go

```diff
@@ -530,10 +530,14 @@ func LoadProjectForVersion(v *Version, id string, shouldSave bool) (ProjectInfo,
 	p := &Project{}
 	// opts empty because project yaml with `include` will not hit this case
 	ctx := context.Background()
-	pp, pc, err := LoadProjectInto(ctx, []byte(v.Config), nil, id, p)
+	pp, err = LoadProjectInto(ctx, []byte(v.Config), nil, id, p)
 	if err != nil {
 		return ProjectInfo{}, errors.Wrap(err, "error loading project")
 	}
+	pc, err := CreateProjectConfig([]byte(v.Config), id)
```

Comment:
Should this be version controlled?

Reply:
Could pass this in as a boolean and then we don't have to look it up for cases where it's not relevant (anywhere we just use .Project)

Reply:
This was the one instance where I was unsure to check for version control since I wasn't sure if we'd want to ad a DB call looking up project ref.  I've added it now.